### ISSUE TITLE
[1st course] : Gradient Descent

### ISSUE THREAD

**Issue by yesjiyoung** [yesjiyoung/2021_Google_ML_Bootcamp#1]
# Lecture name  : 1st course - Neural Networks Basics
## 1.2.1.4 Gradient Descent 
**<편미분과 일반미분의 표기법 차이>** 

- 편미분 (partial derivative) : ∂θ
- 일반 미분 : dθ
![image](https://user-images.githubusercontent.com/51046701/129029438-7f06a67b-668d-4fca-97e6-d6f847bf910e.png)
